scripts/svm/remoteHubPoolPauseDeposits.ts

```diff
@@ -0,0 +1,211 @@
+// This script bridges remote call to pause deposits on Solana Spoke Pool. Required environment:
+// - ETHERS_PROVIDER_URL: Ethereum RPC provider URL.
+// - ETHERS_MNEMONIC: Mnemonic of the wallet that will sign the sending transaction on Ethereum
+// - HUB_POOL_ADDRESS: Hub Pool address
+
+import * as anchor from "@coral-xyz/anchor";
+import { AnchorProvider, BN, Program, web3 } from "@coral-xyz/anchor";
+import { AccountMeta, PublicKey } from "@solana/web3.js";
+import "dotenv/config";
+import { ethers } from "ethers";
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
+import { MessageTransmitter } from "../../target/types/message_transmitter";
+import { SvmSpoke } from "../../target/types/svm_spoke";
+import { decodeMessageHeader, getMessages } from "../../test/svm/cctpHelpers";
+import { HubPool__factory } from "../../typechain";
+import { CIRCLE_IRIS_API_URL_DEVNET, CIRCLE_IRIS_API_URL_MAINNET } from "./utils/constants";
+
+// Set up Solana provider.
+const provider = AnchorProvider.env();
+anchor.setProvider(provider);
+
+// Parse arguments
+const argv = yargs(hideBin(process.argv))
+  .option("chainId", { type: "string", demandOption: true, describe: "Chain ID" })
+  .option("pause", { type: "boolean", demandOption: true, describe: "Pause deposits" })
+  .option("resumeRemoteTx", { type: "string", demandOption: false, describe: "Resume receiving remote tx" }).argv;
+
+async function remoteHubPoolPauseDeposit(): Promise<void> {
+  const resolvedArgv = await argv;
+
+  const chainId = resolvedArgv.chainId;
+  const seed = new BN(0);
+  const resumeRemoteTx = resolvedArgv.resumeRemoteTx;
+  const pause = resolvedArgv.pause;
+
+  // Set up Ethereum provider.
+  if (!process.env.ETHERS_PROVIDER_URL) {
```

Comment:
also this is diffrent to how the rest of the repo does this. either you should should use `CUSTOM_NODE_URL` or use `getNodeUrl` like done in other places